test/integration/cloud_event.ts

```diff
@@ -39,7 +39,7 @@ const TEST_EXTENSIONS = {
 describe('CloudEvent Function', () => {
   let clock: sinon.SinonFakeTimers;
 
-  let receivedCloudEvent: functions.CloudEvent | null;
+  let receivedCloudEvent: functions.CloudEvent<unknown> | null;
```

Comment:
It would be a little funky requiring a `unknown` here. Question: 

- `functions.CloudEvent` (can we do this?)
- `functions.CloudEvent<unknown>`